server/app/views/admin/TranslationFormView.java

```diff
@@ -75,17 +77,17 @@ private ContainerTag renderLanguageLink(
    * Renders a form that allows an admin to enter localized text for an entity's applicant-visible
    * fields.
    */
-  protected ContainerTag renderTranslationForm(
+  protected FormTag renderTranslationForm(
       Http.Request request,
       Locale locale,
       String formAction,
       ImmutableList<FieldWithLabel> formFields) {
-    ContainerTag form =
+    FormTag form =
         form()
             .withMethod("POST")
             .with(makeCsrfTokenInputTag(request))
             .withAction(formAction)
-            .with(each(formFields, FieldWithLabel::getContainer))
+            .with(each(formFields, FieldWithLabel::getInputTag)) // TODO fix this ugliness
```

Comment:
Prefer TODOs using a specific issue number (e.g. the J2HTML issue) so we don't lose context on why the TODO is needed.

Additionally, when we eventually go to close the issue, we can check for any outstanding TODOs prior to considering the work "done".

Reply:
https://github.com/seattle-uat/civiform/issues/2764